src/components/RatLeaderboardTable.js

```diff
@@ -125,16 +119,16 @@ class RatLeaderboardTable extends React.Component {
           filterable: false,
           Cell: ({ value }) => {
             const {
-              codeRed,
-              createdAt,
+              codeRedCount,
+              joinedAt,
               rescueCount,
             } = value.attributes
 
             return (
               <div className="badge-list">
                 <RescueAchievementIcon className="size-32 fixed" rescueCount={rescueCount} />
-                <CodeRedIcon className="size-32 fixed" codeRedCount={safeParseInt(codeRed)} />
-                <FirstYearIcon className="size-32 fixed" createdAt={createdAt} />
+                <CodeRedIcon className="size-32 fixed" codeRedCount={codeRedCount} />
+                <FirstYearIcon className="size-32 fixed" createdAt={joinedAt} />
```

Comment:
Wouldn't it make sense to change `createdAt` to `joinedAt` as well? Seems more legible.

Reply:
yeaaaaaaa I just wasn't too worried about it because we're gonna be ditching most of this someday soon ™